src/Viewer/services/decoder/FourByteClpIrStreamReader.js

```diff
@@ -201,8 +209,7 @@ class FourByteClpIrStreamReader {
             // FIXME: This only supports verbosity levels starting at the 2nd
             //  character of the log type
             if (uint8ArrayContains(this._logtype.getValueUint8Array(), 1,
-                verbosityUint8Array, 0))
-            {
+                verbosityUint8Array, 0)) {
```

Comment:
Accidental change?

Reply:
I guess this is the formatter's change. Should we just keep this change so that it matches with our styling?

Reply:
Let's revert it back and run ESLint again. If it gets formatted the same way, I feel it's okay to submit as-is.

Reply:
yup it gets formmated